email-utils.js

```diff
@@ -78,15 +78,26 @@ const EmailUtils = {
     });
   },
 
-  verifyUrl (subscriber) {
-    return `${AppConstants.SERVER_URL}/user/verify?token=${encodeURIComponent(subscriber.verification_token)}`;
+  utmParams(campaign, content) {
+    return `?utm_source=fx-monitor-emails&utm_medium=email&utm_campaign=${campaign}&utm_content=${content}`;
```

Comment:
Because `utmParams` returns `?utm...`, it makes `getVerificationUrl` return http://localhost:6060/user/unsubscribe?token=null&hash=12eeb998fe0c7600b3a3aacf518aa760b1dbd775?utm_source=fx-monitor-emails&utm_medium=email&utm_campaign=unsubscribe&utm_content=report

Note that `?utm_source` is appended after `&hash=...` (and that `token=null`?)

The `utmParams` function should combine the utm params with any existing url params. In [the new FxA oauth `init` code](https://github.com/mozilla/blurts-server/blob/5932394e2ba15439a6f41dba01a1c4cf7a9b51f5/controllers/oauth.js#L38-L50), [we use `URLSearchParams` and `append` to do this](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). Could do the same thing here.